example-junit/pom.xml

```diff
@@ -74,12 +103,6 @@
                         -javaagent:"${settings.localRepository}/org/aspectj/aspectjweaver/${aspectj.version}/aspectjweaver-${aspectj.version}.jar"
                         -javaagent:${com.nordstrom.tools:junit-foundation:jar}
                     </argLine>
-                    <properties>
-                        <property>
-                            <name>listener</name>
-                            <value>com.nordstrom.automation.junit.RunAnnouncer</value>
-                        </property>
-                    </properties>
```

Comment:
This listener specification was the source of the NullPointerException failures reported in [JUnit Foundation issue #62](https://github.com/Nordstrom/JUnit-Foundation/issues/62).